src/System.IO.Compression/tests/System.IO.Compression.Tests.csproj

```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <ProjectGuid>{17DA7FB5-4370-4385-9A02-FFEF9F482903}</ProjectGuid>
     <DefineConstants Condition="'$(TargetGroup)' == 'netcoreapp' or '$(TargetGroup)' == 'uap'">$(DefineConstants);netcoreapp;STREAM_MEMORY_OVERLOADS_AVAILABLE</DefineConstants>
```

Comment:
You can remove the DefineConstants here and all `#if netcoreapp` and `#if STREAM_MEMORY_OVERLOADS_AVAILABLE` now as we don't compile against netstandard anymore.

Reply:
I haven't found any usage of `netcoreapp`

`STREAM_MEMORY_OVERLOADS_AVAILABLE` is used in `Common/tests/../CompressionStreamunitTestBase.cs` which is also included from System.IO.Compression.Brotli.Tests. Should I also remove netfx from that project, or should we keep this limited in scope?

Reply:
System.IO.Compression.Brotli.Tests doesn't have a netfx configuration? It seems someone forgot to clean that project up. You can remove the DefineConstants there too and please remove the uap configuration in `Configurations` there as it seems to be a leftover.

Reply:
Maybe this clean up should happen as a separate PR, to unblock the scoped fix for #29735.